lagunita/lms/templates/static_templates/honor.html

```diff
@@ -1,51 +1,24 @@
 <%page expression_filter="h"/>
 <%!
   from django.utils.translation import ugettext as _
+  from edxmako.shortcuts import marketing_link
 %>
 <%inherit file="../main.html" />
-<%namespace name='theme_settings' file='/theme-settings.html'/>
 <%block name="pagetitle">${_('Honor Code')}</%block>
-<%
-  theme_vars = theme_settings.get_theme_vars()
-  effective_date = theme_vars['effective_date']
-%>
+<%block name="headextra">
+    <%!
+      from edxmako.shortcuts import marketing_link
+    %>
+    <meta charset="UTF-8">
+    <meta http-equiv="refresh" content="0; url=${marketing_link('HONOR')}">
```

Comment:
@caesar2164 Can't you stick this in variable above (right after the import) instead of performing the lookup multiple times?